pages/docs/concepts/asyncapi-document/payload-schema.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Payload Schema
+weight: 239
```

Comment:
Either this page weight number is not sufficiently high, or something needs to be fixed... do you notice in your preview link that the `Overview` page shows incorrectly AFTER this new document?

BTW, moving forward, let's remember to always review your work via the generated Netlify preview link generated in each PR. Ideally, you should have noticed this before I did 😄

<img width="300" alt="Screen Shot 2023-08-31 at 6 20 29 PM" src="https://github.com/asyncapi/website/assets/19964402/5932e56b-57f8-40c3-a0d4-63eb2c273bad">